site/documentation/content/admin-guide/command-router-config.md

```diff
@@ -0,0 +1,154 @@
+
+The secure port will mostly be required for production scenarios. However, it might be desirable to expose a non-TLS secured port instead, e.g. for testing purposes. In any case, the non-secure port needs to be explicitly enabled either by
+
+- explicitly setting `HONO_COMMANDROUTER_AMQP_INSECURE_PORT` to a valid port number, or by
+- implicitly configuring the default AMQP port (5672) by simply setting `HONO_COMMANDROUTER_AMQP_INSECURE_PORT_ENABLED` to `true`.
+
+The server issues a warning on the console if `HONO_COMMANDROUTER_AMQP_INSECURE_PORT` is set to the default secure AMQP port (5671).
+
+The `HONO_COMMANDROUTER_AMQP_INSECURE_PORT_BIND_ADDRESS` variable can be used to specify the network interface that the port should be exposed on. By default the port is bound to the *loopback device* only, i.e. the port will only be accessible from the local host. This variable might be used to e.g. expose the non-TLS secured port on a local interface only, thus providing easy access from within the local network, while still requiring encrypted communication when accessed from the outside over public network infrastructure.
+
+Setting this variable to `0.0.0.0` will let the port being bound to **all** network interfaces (be careful not to expose the port unintentionally to the outside world).
+
+### Dual Port
+ 
+In test setups and some production scenarios Hono server may be configured to open one secure **and** one insecure port at the same time.
+ 
+This is achieved by configuring both ports correctly (see above). The server will fail to start if both ports are configured to use the same port number.
+
+Since the secure port may need different visibility in the network setup compared to the secure port, it has its own binding address `HONO_COMMANDROUTER_AMQP_INSECURE_PORT_BIND_ADDRESS`. 
+This can be used to narrow the visibility of the insecure port to a local network e.g., while the secure port may be visible worldwide. 
+
+### Ephemeral Ports
+
+Both the secure as well as the insecure port numbers may be explicitly set to `0`. The Command Router component will then use arbitrary (unused) port numbers determined by the operating system during startup.
+
+## Data Grid Connection Configuration
+
+The Command Router component requires either an embedded cache or a remote
+data grid, using the Infinispan Hotrod protocol to store device information.
+
+The following table provides an overview of the configuration variables and corresponding command line options for configuring the common aspects of the service:
+
+| Environment Variable<br>Command Line Option | Mandatory | Default | Description                                                             |
+| :------------------------------------------ | :-------: | :------ | :-----------------------------------------------------------------------|
+| `HONO_DEVICECONNECTION_COMMON_CACHENAME`<br>`--hono.deviceConnection.common.cacheName` | no | `device-connection` | The name of the cache |
```

Comment:
`_DEVICECONNECTION_` ?

Reply:
I have used that deliberately, combining configuration data used by the `(CacheBased)DeviceConnectionInfo` class (which will probably still be there even when the DeviceConnection API has been removed).

Another option would be using "commandRouter.cache" instead (as in `hono.commandRouter.cache.common.cacheName` for example). I'd be fine with that as well. WDYT?

Reply:
FMPOV it would be rather confusing to have configuration properties with two different prefixes for configuring the command router service, don't you think?

Reply:
Well, configuring access to the (possibly external) cache *could* be seen as analogous to the configuration for the other external services (registration, credentials, etc), having their own prefix.
But as it's no defined external Hono *service*, I see that the "hono.commandRouter.cache" prefix may be better.

I've added commits now changing the prefix in the Command Router component and in the above documentation and also changing the default cache name.